src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/392b3c8cfd.js" crossorigin="anonymous"></script>
+    <title>Javascript Capstone Project</title>
+</head>
+<body>
+    <header>
+        <nav class="nav">
+            <i class="fa-solid fa-film movie-icon" ></i>
+            <ul>
+                <li> <a  class="movie" href="#"></a></li>
+                <li><a href="#">Comment<i class="fa-regular fa-comment"></i></a></li>
+                <li><a href="#">Like<i class="fa-solid fa-heart"></i></a></li>
+              </ul>
+        </nav>
```

Comment:
- [ ] Hi, The navigation bar belongs only to the home page, as we follow the wireframe provided by the project requirements. Can you remove the navigation bar from the pop-up?
- Your page
![Capture1](https://github.com/tatouroberto/javascriptCapstone/assets/63972670/5a9d654c-d45b-4229-b72e-8fb3954d8956)

- Original

![1](https://github.com/tatouroberto/javascriptCapstone/assets/63972670/a92c79d1-3dba-4b50-ad7d-e101156eac8c)
NB: Follow the red mark I drew on your picture to understand what I am talking about.

Reply:
Fixed .